app/controllers/installer_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# InstallerController shows the main page.
+class InstallerController < ApplicationController
```

Comment:
Moreover, the following methods look like pages from the ConfigController. Maybe both classes can be merged ?